Dockerfile.moth

```diff
@@ -7,4 +7,6 @@ RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' -o /mo
 FROM scratch
 COPY --from=builder /mothd /mothd
 COPY theme /theme
+COPY LICENSE.md /LICENSE
```

Comment:
Do we need to do this, or would it be better to expose this someplace less obscure (like when you run the binary)? I can honestly say it's never even occured to me to check inside a Docker image for a license, but I can totes see the logic here.

Reply:
I don't think that there's a strong reason *not* to, except to save a few K in the Docker image. Where would make more sense to place this file?